.browserslistrc

```diff
@@ -0,0 +1,6 @@
+# Browsers that we support
+
+last 1 major versions
```

Comment:
In your original description of the PR, you mentioned `last 2 major versions`. Any reason in particular for the change?

We should also support Firefox ESR. When answering support requests, I've seen multiple folks running that in institutional and corporate environments.

Reply:
I did 2 versions